src/transformers/models/opt/modeling_tf_opt.py

```diff
@@ -645,6 +645,15 @@ def call(
 
         if attention_mask is None:
             attention_mask = tf.ones(inputs_embeds.shape[:2], dtype=tf.bool)
+        else:
+            tf.debugging.assert_less_equal(
```

Comment:
Why is it `assert_less_equal` here and not `assert_equal`?

Reply:
It should be `assert_equal` indeed (the exception was different in the previous commit, and I forgot to update the condition) -- thank you for noticing it!